min/lib/CSSmin.php

```diff
@@ -765,7 +765,9 @@ private function str_slice($str, $start = 0, $end = FALSE)
     private function normalize_int($size)
     {
         if (is_string($size)) {
-            switch (substr($size, -1)) {
+            $letter = substr($size, -1);
+            $size = intval($size);
```

Comment:
Worth changing as a cs rule, if not as a micro optimization. The only reason intval hasn't been removed from the php core is because it can be used with array maps, and the like.